jac-uk/admin
# #2534 Fix broken layout

## What's included?
Fixes the layout of:
 - [x] Tasks
 - [x] Reports
 - [x] Stages

Note: I have removed `govuk-grid-row` from the top-level templates for Tasks, Reports and Stages. It is easier to manage if grid row and column code is together in the same file. 

Closes #2534

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Check all views in Admin and update the parent ticket with any additional views that need fixing

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Before (content too wide)
<img width="1232" alt="image" src="https://github.com/user-attachments/assets/b4a0dcdc-5992-4993-abea-27c2082cefdc">


After (content same width as header)
<img width="1187" alt="image" src="https://github.com/user-attachments/assets/4263e882-f209-4ccf-b497-1ccff323d528">


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
